### ISSUE TITLE
MBR Corruption on Windows?

### ISSUE THREAD

**Issue by Jekteir** [ethereum/go-ethereum#14415]
#### System information

Geth version: https://gethstore.blob.core.windows.net/builds/geth-windows-amd64-1.6.0-facc47cb.exe as linked to from https://ethereum.github.io/go-ethereum/downloads/
OS & Version: Windows 10 Pro x64

#### Expected behaviour

geth --rpc --fast --cache=1024 downloads the blockchain safely overnight.

#### Actual behaviour

In the morning, I activate my monitors and get a "cannot boot"-type error. I go into the BIOS settings and discover that only 2 of my 4 hard drives are visible. Neither the system SSD drive (C) on which Windows is stored, nor the hard disk data drive on which the geth .exe file was running, are visible in the drives list in the BIOS. I know this because the C drive is a 120GB SSD and the big data drive is a 4TB drive, and the only visible 2 drives in the BIOS list were 300GB and 1TB -- my scratch drives.

#### Steps to reproduce the behaviour

1. Download geth from the link above.
2. Run it with the --rpc --fast --cache=1024 options and leave the PC running overnight, with the screen locked.
3. Wake up to find it bricked :(

When I get home from work tonight I'm going to experiment with booting to a Windows 10 repair USB key and seeing what it finds. And I may try taking out the drive(s) and plugging them into another PC. But I can't even see 2 of my hard drives from BIOS, which is scary. Could there be a virus in the geth .exe I listed above? I downloaded it last night. If not, could running out of space on C while downloading the blockchain (absolutely possible this could have happened, if the blockchain is saved to C) have corrupted my hard drive(s) somehow?

Straight hard drive failure seems very unlikely given 2 different drives, one physical and one SSD, are both no longer visible.

**Comment by karalabe** [comment]
I really cannot see how such a thing could be caused by Geth. We don't have any more access to your file system than any other program. As for viruses, I'm quite positive we don't have those, but feel free to submit the exe to virus total if it makes you feel more convinced.

**Comment by Jekteir** [comment]
Hi Karalabe, thanks for the response! I already ran the virus scan and detected nothing, and I figured that will all the commits and visibility on this tool, virus was unlikely (unless someone hacked the download server, I guess). Looks like that wasn't the problem :)

When I got home in the evening I messed more with the computer and established that if I moved my primary drive to a different SATA controller chip (there are 2 on my mobo) and disconnected all other drives, I could boot into Windows.

I don't know if it was Geth's performance intensive nature or simply a coincidence of timing, but right now I believe that one of my disk controller chips has stopped working. I disconnected my 2 scratch drives and moved my system and data disk over to the other controller and have been able to use the PC fine since then. I didn't notice any bad capacitors on my motherboard but I'll have to dig in more and confirm. I'll close out this issue. Cheers!
